plugins/woocommerce-blocks/tests/e2e/playwright-utils/test.ts

```diff
@@ -154,6 +160,15 @@ const test = base.extend<
 	storeApiUtils: async ( { requestUtils }, use ) => {
 		await use( new StoreApiUtils( requestUtils ) );
 	},
+	shippingUtils: async ( { page }, use ) => {
+		await use( new ShippingUtils( page ) );
```

Comment:
Since this uses `admin` I'm wondering if it's possible to add `this.admin` to the args? Might not work but we should try it out. Same for `localPickupUtils`.

Reply:
See https://github.com/woocommerce/woocommerce/pull/44108#discussion_r1469451202